### DISCUSSION TITLE
Please tell me your favorite Onigiri!

### DISCUSSION THREAD

**Discussion by ghost** [NAIST-SE/AttractingContributors#176] [General]
https://github.com/hirokiic1/AttractingContributors-Onigiri
Please tell me your favorite Onigiri.
Thank you!

**Comment by ElefunAi** [comment]
I have added Onigiri list!
Please check and help me!

https://github.com/ElefunAi/AttractingContributors-texteditor

**Reply by ghost** [reply]
I added too.
Thank you! 😄
